src/twisted/test/test_application.py

```diff
@@ -889,3 +896,54 @@ def install():
                                ['--reactor', 'fakereactortest', 'subcommand'])
         self.assertIn(message, e.args[0])
         self.assertIn("help-reactors", e.args[0])
+
+
+
+class HelpReactorsTests(unittest.TestCase):
+    """
+    --help-reactors lists the available reactors
+    """
+    def setUp(self):
```

Comment:
Also, this TestCase is a poster-child for testtools-style `expectThat` assertions. :)